app/admin/dashboard.rb

```diff
@@ -33,6 +33,21 @@
             end
           end
         end
+        # Chartkick charts here
+        panel "Adoptions Over Time" do
+          line_chart Adoption.all.group_by { |adp| adp.created_at.beginning_of_month }
```

Comment:
Suggestion: We can use this https://github.com/ankane/groupdate gem to simplify the logic.

But, I remember that you don't want to add a gem if we can do without it. That is why it's the pure Rails right now.

Reply:
this looks good as is 👍